src/perform_step/firk_perform_step.jl

```diff
@@ -749,3 +775,528 @@ end
     integrator.stats.nf += 1
     return
 end
+
+@muladd function perform_step!(integrator, cache::RadauIIA7ConstantCache,
+    repeat_step = false)
+    @unpack t, dt, uprev, u, f, p = integrator
+    @unpack T11, T12, T13, T14, T15, T21, T22, T23, T24, T25, T31, T32, T33, T34, T35, T41, T42, T43, T44, T45, T51 = cache.tab #= T52 = 1, T53 = 0, T54 = 1, T55 = 0=#
+    @unpack TI11, TI12, TI13, TI14, TI15, TI21, TI22, TI23, TI24, TI25, TI31, TI32, TI33,TI34, TI35, TI41, TI42, TI43, TI44, TI45, TI51, TI52, TI53, TI54, TI55 = cache.tab
+    @unpack c1, c2, c3, c4, γ, α1, β1, α2, β2, e1, e2, e3, e4, e5 = cache.tab
+    @unpack κ, cont1, cont2, cont3, cont4 = cache
+    @unpack internalnorm, abstol, reltol, adaptive = integrator.opts
+    alg = unwrap_alg(integrator, true)
+    @unpack maxiters = alg
+    mass_matrix = integrator.f.mass_matrix
+
+    # precalculations
+    rtol = @.. broadcast=false reltol^(2 / 3)/10
+    atol = @.. broadcast=false rtol*(abstol / reltol)
```

Comment:
where do these come from? that seems very odd...

Reply:
they're the same values used in the other Radau methods so I just copied them